grafana-plugin/src/models/alertgroup/alertgroup.ts

```diff
@@ -294,21 +280,30 @@ export class AlertGroupStore extends BaseStore {
     // @ts-ignore
     this.alerts = new Map<number, Alert>([...this.alerts, ...newAlerts]);
 
+    this.incidentsItemsPerPage = page_size;
```

Comment:
do we really need to store this in alertgroupstore as a separate field? can we get it as `getSearchResult().page_size`?

Reply:
I guess you're right, let me try it :thinking:

Reply:
weird but ok